pulseguardian/config.py

```diff
@@ -42,6 +42,7 @@
 warn_queue_size = int(os.getenv('WARN_QUEUE_SIZE', 2000))
 del_queue_size = int(os.getenv('DEL_QUEUE_SIZE', 8000))
 polling_interval = int(os.getenv('POLLING_INTERVAL', 5))
+polling_max_interval = int(os.getenv('POLLING_INTERVAL', 300))
```

Comment:
This environment variable should be called something different, e.g. POLLING_MAX_INTERVAL.